pubspec.yaml

```diff
@@ -24,8 +24,8 @@ dependencies:
   platform_detect: ^1.3.2
   quiver: ">=0.21.4 <0.26.0"
 dev_dependencies:
-  build_runner: ^0.5.0
-  built_value_generator: ^4.2.0
+  build_runner: ^0.6.0
+  built_value_generator: ^5.1.3
```

Comment:
@jacehensley-wf should we leave the lower bound intact for `build_runner` / `built_value_generator`?

Reply:
If I don't update these I cannot pull in the newer version of `built_value` locally. Since it's just a dev_dependency it's not a big issue to just bump these instead of widening.